.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+Before submitting your PR, check that you've:
+
+- [ ] Documented new APIs with [Haddock markup](https://www.haskell.org/haddock/doc/html/index.html)
+- [ ] Added [`@since` declarations](http://haskell-haddock.readthedocs.io/en/latest/markup.html#since) to the Haddock
+
+After submitting your PR:
+
+- [ ] Update the Changelog.md file with a link to your PR
```

Comment:
I don't quite understand how it is supposed to work. I mean, a contributor fills out this form before submitting a PR, right?..

In my experience asking contributors to amend changelog is a source of merge conflicts, when there are several PR in flight. Contributors are also generally bad at following the same style of entries. IMO it's easier for maintainers to update changelog before release than chase everyone to edit and rebase.

Reply:
It works perfectly fine with `persistent`. It occasionally happens that you need to resolve a conflict, but it's easy enough.

Most contributors copy/paste the previous changelog item and fill in the relevant details. We could provide a common format in this file, to make it easier.

Reply:
OK, let's give it a try. But how is it supposed to work? Once a PR is created, should a contributor push an additional commit and edit message to cross out this item?

Reply:
yup!